tests/DependencyInjection/AwsExtensionTest.php

```diff
@@ -60,12 +34,17 @@ public function all_web_services_in_sdk_manifest_should_be_accessible_as_contain
      *
      * @param string $serviceName
      * @param string $serviceRegion
+     * @param AwsClient $clientClassName
      */
     public function sdk_config_should_be_passed_directly_to_the_constructor_and_resolved_by_the_sdk(
         $serviceName,
-        $serviceRegion
+        $serviceRegion,
+        $clientClassName
     ) {
-        $service = $this->container->get($serviceName);
+
+        $this->args['region'] = $serviceRegion;
+
+        $service = new $clientClassName($this->args);
```

Comment:
I'm not sure this test is still useful. SDK client construction is tested in the SDK -- tests in this package should exercise the integration with Symfony instead of SDK internals.